lib/net.js

```diff
@@ -1647,6 +1647,7 @@ function onconnection(err, clientHandle) {
   }
 
   if (self.maxConnections && self._connections >= self.maxConnections) {
+    self.emit('drop');
```

Comment:
How would you use this event? To count dropped connections? It does not seem very useful without arguments.

Reply:
> How would you use this event? To count dropped connections? It does not seem very useful without arguments.

Yes, count the dropped connections. I have added the address info as arguments of drop event, please review again, thanks.